import_data/tasks/tasks.py

```diff
@@ -42,14 +44,37 @@ def _open_sql_connection(ctx, db):
 
 
 def _execute_sql(ctx, sql, db=None, additional_options=""):
-    query = f'psql -Xq -h {ctx.pg.host} -p {ctx.pg.port} -U {ctx.pg.user} -c "{sql}" {additional_options}'
+    query = f'psql -Xq -h {ctx.pg.host} -p {ctx.pg.port} -U {ctx.pg.user} {additional_options}'
     if db is not None:
         query += f" -d {db}"
     return ctx.run(
         query,
-        env={"PGPASSWORD": ctx.pg.password}
+        in_stream=StringIO(sql),
```

Comment:
it seems that invoke consumes this stream very slowly, possibly because of its default value for input_sleep: http://docs.pyinvoke.org/en/1.2/api/runners.html#invoke.runners.Runner.input_sleep

Maybe we would be easier to stick to `psql -f` option and write a temporary file if necessary.